report/renderer/util.js

```diff
@@ -98,6 +98,30 @@ class Util {
             }
           }
         }
+
+        // Circa 10.0, table items were refactored.
+        if (audit.details.type === 'table') {
+          for (const heading of audit.details.headings) {
+            /** @type {{itemType: LH.Audit.Details.ItemValueType|undefined, text: string|undefined}} */
+            // @ts-expect-error
+            const {itemType, text} = heading;
+            if (itemType !== undefined) heading.valueType = itemType;
+            if (text !== undefined) heading.label = text;
+
+            // @ts-expect-error
+            delete heading.itemType;
+            // @ts-expect-error
+            delete heading.text;
```

Comment:
can you make these deletes conditional as well? like 
```js
if (itemType !== undefined) {
  heading.valueType = itemType;
  delete heading.itemType;
}
if (text !== undefined) {
  heading.label = text;
  delete heading.text;
}
```
that way up-to-date LHRs aren't touched (not sure if there's a typical JS engine fast path for delete of non-existent properties not altering the object maps, etc, but better to be stingy with `delete` regardless)